src/components/Modal/ProductModal/ProductModal.tsx

```diff
@@ -43,7 +92,15 @@ const ProductModal = () => {
           }
         </div>
       </div>
-      <Recommended currentProductName={name} />
+      {
+        data[0] === 'no products'?
+          ''
+          :
+          data.length === 0 ?
+            'Loading...'
+            :
+            <Recommended products={data}/>
```

Comment:
Instead of `Loading...` there should be a mini loader (like a rotating circle for example, ask better Valery)